From 7d5b814d898f4c70eb0b2424965ed1b7e347d424 Mon Sep 17 00:00:00 2001
From: Ran Wang <ran.wang_1@nxp.com>
Date: Wed, 13 Jun 2018 16:39:53 +0800
Subject: [PATCH 065/107] Revert "EHCI: Fix endian access issue on EHCI intinalization"

There was an EHCI endian accessor issue. Now it's fixed by
commit 9829ce2ff25c ("usb: ehci: Fix accessors for big-endian platforms and descriptors")
Revert commit b1336202d1f3 ("EHCI: Fix endian access issue on EHCI intinalization")
due to it fix that issue in a wrong way. Actually, on T & P serial
platforms, USB EHCI register use little endian.

Signed-off-by: Ran Wang <ran.wang_1@nxp.com>
---
 include/configs/P2041RDB.h   |    1 -
 include/configs/T102xRDB.h   |    1 -
 include/configs/T208xRDB.h   |    1 -
 include/configs/T4240RDB.h   |    1 -
 include/configs/corenet_ds.h |    1 -
 5 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/include/configs/P2041RDB.h b/include/configs/P2041RDB.h
index afa6888..3bfc35c 100644
--- a/include/configs/P2041RDB.h
+++ b/include/configs/P2041RDB.h
@@ -569,7 +569,6 @@ unsigned long get_board_sys_clk(unsigned long dummy);
 
 #if defined(CONFIG_HAS_FSL_DR_USB) || defined(CONFIG_HAS_FSL_MPH_USB)
 #define CONFIG_USB_EHCI_FSL
-#define CONFIG_EHCI_DESC_BIG_ENDIAN
 #define CONFIG_EHCI_HCD_INIT_AFTER_RESET
 #endif
 
diff --git a/include/configs/T102xRDB.h b/include/configs/T102xRDB.h
index 839a353..3d3b46a 100644
--- a/include/configs/T102xRDB.h
+++ b/include/configs/T102xRDB.h
@@ -629,7 +629,6 @@ unsigned long get_board_ddr_clk(void);
 
 #ifdef CONFIG_HAS_FSL_DR_USB
 #define CONFIG_USB_EHCI_FSL
-#define CONFIG_EHCI_DESC_BIG_ENDIAN
 #define CONFIG_EHCI_HCD_INIT_AFTER_RESET
 #endif
 
diff --git a/include/configs/T208xRDB.h b/include/configs/T208xRDB.h
index 084c29e..48eb1d0 100644
--- a/include/configs/T208xRDB.h
+++ b/include/configs/T208xRDB.h
@@ -636,7 +636,6 @@ unsigned long get_board_ddr_clk(void);
  */
 #ifdef CONFIG_USB_EHCI_HCD
 #define CONFIG_USB_EHCI_FSL
-#define CONFIG_EHCI_DESC_BIG_ENDIAN
 #define CONFIG_EHCI_HCD_INIT_AFTER_RESET
 #define CONFIG_HAS_FSL_DR_USB
 #endif
diff --git a/include/configs/T4240RDB.h b/include/configs/T4240RDB.h
index a71cb08..21d100a 100644
--- a/include/configs/T4240RDB.h
+++ b/include/configs/T4240RDB.h
@@ -641,7 +641,6 @@ unsigned long get_board_ddr_clk(void);
 * USB
 */
 #define CONFIG_USB_EHCI_FSL
-#define CONFIG_EHCI_DESC_BIG_ENDIAN
 #define CONFIG_EHCI_HCD_INIT_AFTER_RESET
 #define CONFIG_HAS_FSL_DR_USB
 
diff --git a/include/configs/corenet_ds.h b/include/configs/corenet_ds.h
index 6f73aeb..b6688b0 100644
--- a/include/configs/corenet_ds.h
+++ b/include/configs/corenet_ds.h
@@ -578,7 +578,6 @@
 
 #if defined(CONFIG_HAS_FSL_DR_USB) || defined(CONFIG_HAS_FSL_MPH_USB)
 #define CONFIG_USB_EHCI_FSL
-#define CONFIG_EHCI_DESC_BIG_ENDIAN
 #define CONFIG_EHCI_HCD_INIT_AFTER_RESET
 #endif
 
-- 
1.7.1

